compiler/cli/src/org/jetbrains/kotlin/cli/jvm/plugins/DependencyShadingPluginClassLoader.kt

```diff
@@ -0,0 +1,68 @@
+package org.jetbrains.kotlin.cli.jvm.plugins
```

Comment:
Please reformat the file ("Reformat file..." action) and add copyright ("Update Copyright...")